components/input/docs/partials/api.md

```diff
@@ -193,6 +212,25 @@ The `min` attribute should be used in combination with the `setCustomValidityRan
 
 </auro-accordion>
 
+### Min Length <a name="minLength"></a>
+
+Use the `minlength` attribute to control the length of the input entered.
+
+The `minLength` attribute should be used in combination with the `setCustomValidityTooShort` attribute to define help text used when the length of the input is not long enough.
```

Comment:
Same comment as above for maxLength.

Reply:
I added this to follow suit with what we are currently doing on the docsite. You can see us suggesting them to use the `setCustomValidity` states under the "Pattern", "Max" and "Min" examples on the API page.

Reply:
I'm all for the _suggesting_. I think the language "... should be used ..." is problematic because they don't actually have to do it.

Maybe something like....

```suggestion
The input will display a default validation error message associated with this attribute when validation fails. Each browser client defines it's own localized message. The `setCustomValidityTooShort` attribute may optionally be used in combination with the `minLength` attribute to define custom help text used when the length of the input is not long enough.
```

If you don't want to make that sort of change just open a doc ticket for some future tech debt we can review as a team.